Dispatcher/Controller/UserPageViewController.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+class UserPageViewController: UIViewController {
+    let navigationBar = NavigationBar()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationBar.delegate = self
+        navigationBar.setupNavigationBar(for: self)
+    }
+    
+}
+
+extension UserPageViewController:HeaderDelegate {
```

Comment:
please add space to UserPageViewController: HeaderDelegate

its harder to read when its cramped together